CHANGELOG.md

```diff
@@ -19,6 +19,46 @@ under the License.
 
 ## Change Log
 
+### 2.0.1
+
+### 2.0 (Fri Jul 22 13:56:33 2022 -0400)
```

Comment:
can you combine lines 22 and 24 and have it say 2.0.1?

Reply:
fixed